src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlValueConverter.cs

```diff
@@ -268,6 +269,9 @@ protected XmlBaseConverter(XmlTypeCode typeCode)
             _typeCode = typeCode;
         }
 
+
+        [UnconditionalSuppressMessage("AotAnalysis", "IL3050:AotUnfriendlyApi",
+            Justification = "All types that are instantiated with this method are used elsewhere throughout System.Private.Xml.")]
```

Comment:
This is complaining about array types. Are all possible array types that this can create really guaranteed to be used by something else?

Reply:
I believe so, but I'll remove the suppression and propagate this up to callers to move the suppression closer to all places where the System.Type that's the element type is determined.

Reply:
It may be best to submit the libraries fixes in a separate PR so that they get proper scrutiny.

Reply:
I've revised the message here after some investigation. I'll split the commits into a different PR as well as pushing here to make sure there aren't any other issues.